README.md

```diff
@@ -7,7 +7,7 @@ A docker image with the tools needed to compile and test [forte-music/core]. It
 includes:
 
 * a node environment with yarn
-* a nightly rust toolchain (2018-01-20) along with clippy and rustfmt
+* a stable rust toolchain along with clippy and rustfmt
```

Comment:
This isn't consistent with what is actually installed. Two versions of rust are installed. The stable version and a nightly version with clippy. Change this.

Reply:
I had changes staged which remove the nightly toolchain, but haven't pushed yet.

Reply:
Are you planning to push or not? Is this PR ready?